tests/test_utils.py

```diff
@@ -0,0 +1,157 @@
+"""Tests for the 'utils' module."""
+
+import json
+from unittest import TestCase, mock
+from utils import execute_gremlin_dsl, select_latest_version,\
+    get_response_data, check_license_conflict
+
+
+@mock.patch("utils.get_session_retry")
+def test_check_license_conflict(mocker):
+    """Test the function check_license_conflict."""
+    payload = {"a": "b"}
+    mocker.return_value = MockedSession("lic_true")
+    out = check_license_conflict(payload)
+    print(out)
+    assert out is "false"
+
+    mocker.return_value = MockedSession("lic_false")
+    out = check_license_conflict(payload)
+    print(out)
+    assert out is "true"
+
+
+def test_execute_gremlin_dsl2():
+    """Test the function execute_gremlin_dsl."""
+    query_str = "g.V().has('ecosystem', eco).has('name',pkg).valueMap()"
+    payload = {
+        'gremlin': query_str,
+        'bindings': {
+            'eco': 'maven',
+            'pkg': 'io.vertx:vertx-web'
+        }
+    }
+    out = execute_gremlin_dsl(payload)
```

Comment:
how it works on CI w/o the gremlin setup?

Reply:
This is just a function, actual connection happens via the function get_session_retry, which is used to make the post call. Here in this case, we are mocking this up

Reply:
As for this particular test case, there wont be any connection and by default it should return None via exception

Reply:
oh ok, TY